### PR TITLE
Moving FAQ - Finding BAM dataset identifiers

### PR DESCRIPTION
**Pull request by Melkeb**
This FAQ is part of [Mismatched Chromosome identifiers (and how to avoid them)](https://galaxyproject.org/support/chrom-identifiers/)

### CODE REVIEW

File: faqs/galaxy/datasets_BAM_dataset_identifiers.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Finding BAM dataset identifiers
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+Quickly learn what the identifiers are in any **BAM** dataset that is the result from mapping.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_BAM_dataset_identifiers.md:10]: this looks to me like a something that could go to the "description" field in the metadata? See [this example](https://github.com/galaxyproject/training-material/blob/main/faqs/galaxy/analysis_troubleshooting.md).

File: faqs/galaxy/datasets_BAM_dataset_identifiers.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Note: This will *not* work for "sequence-only" `bam` datasets, as these usually have no header.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_BAM_dataset_identifiers.md:12]: Number 4 is also a note, would it make sense to have them together at the end?

File: faqs/galaxy/datasets_BAM_dataset_identifiers.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Run **Samtools: IdxStats** on the aligned data (`bam` dataset).
+2. The "index header" chromosome names and lengths will be listed in the output (along with read counts).
+3. Compare the chromosome identifiers to the chromosome (aka "chrom") field in all other inputs: VCF, GTF, GFF(3), BED, Interval, etc.
+4. Note: The orignal mapping target may have been a built-in genome index, custom genome (transcriptome, exome, other) -- the same `bam` data will still be summarized.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_BAM_dataset_identifiers.md:17]:
```suggestion
4. Note: The original mapping target may have been a built-in genome index, custom genome (transcriptome, exome, other) -- the same `bam` data will still be summarized.
```

File: faqs/galaxy/datasets_BAM_dataset_identifiers.md (hunk 2 above)
Comment (Melkeb) [reply faqs/galaxy/datasets_BAM_dataset_identifiers.md]: Yes it does. Thanks for the comments.

**Review by jennaj — APPROVED** [review]
Excellent, thank you!

**Pull request merged**